crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -45,7 +45,7 @@ use subspace_core_primitives::{BlockNumber, SegmentHeader};
 /// Ideally, we'd decouple pruning from finalization, but it may require invasive changes in
 /// Substrate and is not worth it right now.
 /// https://github.com/paritytech/substrate/discussions/14359
-pub(crate) const FINALIZATION_DEPTH_IN_SEGMENTS: usize = 5;
+pub(crate) const FINALIZATION_DEPTH_IN_SEGMENTS: usize = 1;
```

Comment:
This was used for debugging and should not be committed

Reply:
Sorry, reverted